### PR TITLE
add 1_task ver. 1.0

### PR DESCRIPTION
**Pull request by AlexKisialiou**

### CODE REVIEW

File: 1_task/1_task.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+/**
+ * Created by Alex on 17.03.2017.
+ */
+
+function addition(a, b) {
+    return +a + +b;
```
Comment (NikitaSedyx) [inline 1_task/1_task.js]: `a + b` please. `+a + +b` makes another sense

File: 1_task/1_task.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+function subtraction(a, b) {
+    return a - b;
```
Comment (NikitaSedyx) [inline 1_task/1_task.js]: could you use 2 spaces everywhere?

File: 1_task/1_task.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    return a * b;
+}
+
+function division(a, b) {
+    return a / b;
+}
+
+function integerDivision(a, b) {
+    var fraction = division(a, b);
+    return  fraction - (fraction % 1) ;
```
Comment (NikitaSedyx) [inline 1_task/1_task.js]: could you read about `parseInt` function?

File: 1_task/1_task.js (hunk 2 above)
Comment (AlexKisialiou) [reply 1_task/1_task.js]: you mean I must use 2 spaces instead 4(it's default doing Idea)?
// bad
function() {
∙∙∙∙var name;
}

// bad
function() {
∙var name;
}

// good
function() {
∙∙var name;
}

**Review by AlexKisialiou — COMMENTED** [review]
create some fix

**Comment by NikitaSedyx** [comment]
you are going to merge to layout, please change target branch to master

File: 1_task/index.html [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
```
Comment (NikitaSedyx) [inline 1_task/index.html]: 2 spaces everywhere

File: 1_task/index.html [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+        <meta charset="UTF-8">
+        <title>1_task</title>
+    </head>
+    <body>
+
```
Comment (NikitaSedyx) [inline 1_task/index.html]: I don't see your script is included

File: 1_task/1_task.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+console.log(addition(false, true));
+console.log(addition(false, {}));
+console.log(addition(1, "one"));
+console.log(addition());
+console.log(addition(1));
+console.log(addition(1, 4, 5));
+console.log(subtraction(1, -10));
+console.log(subtraction(1, "-10"));
+console.log(subtraction("-10", "-10"));
+console.log(multiplication(3, 2));
```
Comment (NikitaSedyx) [inline 1_task/1_task.js]: could you add more `multiplication` and  `division` calls with different params?
f.e. 1 + '2' = '12', but 1 * '2' =2, but 1 * 'ad' = NaN

File: 1_task/1_task.js [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+  return a * b;
+}
+
+function division(a, b) {
+  return a / b;
+}
+
+function integerDivision(a, b) {
+  var fraction = division(a, b);
+  return  parseInt(fraction, 10);
```
Comment (NikitaSedyx) [inline 1_task/1_task.js]: it can be one line
`parseInt(division(a, b), 10)`

**Review by NikitaSedyx — CHANGES_REQUESTED** [review]

File: 1_task/1_task.js (hunk 2 above)
Comment (NikitaSedyx) [reply 1_task/1_task.js]: why do you use idea? it's not js ide
2 spaces is mostly used style for html, js, css

File: 1_task/index.html [hunk 8]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <script src="1_task.js"></script>
```
Comment (NikitaSedyx) [inline 1_task/index.html]: basically it's not a good practise to name files with number
name should say what code does

**Review by NikitaSedyx — APPROVED** [review]
lgtm

**Review by AlexKisialiou — COMMENTED** [review]
rename file

**Pull request merged**